src/Mail/MailService.php

```diff
@@ -62,10 +67,10 @@ public function message($to, string $title, string $body, array $attachments = [
                 continue;
             }
 
-            if ('' == trim($person->getName() ?? $person->getUsername() ?? '')) {
+            if ('' == trim($person->getName() ?? $person->getEmail())) {
```

Comment:
Why was this changed? The intent of this line is to check whether a name representation is available and add that to the e-mail address if it is present. However, an e-mail address is intended to be always present, which results in this check being useless in practice.

Reply:
if we assume the to is a `ContactInterface` (which we already did before this PR. `getUsername` does not exist on it. we could go with `getCanonical`. let me know what you prefer.

Reply:
The intent is that a human readable value is provided, which is not the email. Looking at the underlying types of `Address`, my suggestion would be to replace the complete if/else block by:
```php
$addresses[] = new Address($person->getEmail(), $person->getName() ?? '');
```